### ISSUE TITLE
middle click scrolling on tabs

### ISSUE THREAD

**Issue by shalva97** [minbrowser/min#1630]
- **Min Version:** 1.19.2
- **Operating system:** Windows 10

## Expected Behavior
when I middle click tabs (not in task view) they should close

## Actual Behavior
they do close except if there are lots of tabs and a scroll bar appears. then it triggers scrolling effect

## To Reproduce
open lots of tabs until scrollbar appears then middle click and they will not close

**Comment by lubiedo** [comment]
Tested on macOS Big Sur 11.4 and wasn't able to reproduce the issue.

**Comment by shalva97** [comment]
yes, it only happens on Windows...

**Comment by PalmerAL** [comment]
How does this work in other browsers? Using the middle mouse button to scroll horizontally is standard behavior I think, so this might be intended behavior.

**Comment by shalva97** [comment]
here is the video, as you can see it only triggers scroll both in task view and on  tabs.... I dont know if  this is intended but on Linux it didnot scroll at all, which is what I also expect on Windows. 

what do you mean by standard behavior? for me it is -1 useful button on my mouse wasted

https://user-images.githubusercontent.com/22417494/121806454-f7b9ae80-cc60-11eb-8177-30300b7ef798.mp4

**Comment by lubiedo** [comment]
@shalva97 To delete tabs just scroll up or use Ctrl/Meta + W.

**Comment by shalva97** [comment]
@lubiedo did not you read the issue? I cannot close tabs with ctrl + w in  task view nor I can scroll them.

This is happening only on Windows, have you tested it on there?

**Comment by PalmerAL** [comment]
Yeah, there is an actual issue here. Other browser support closing tabs by clicking with the middle mouse button; that works in Min sometimes, but not it the tabs list is scrollable.
